README.md

```diff
@@ -1574,6 +1574,7 @@ Update Time, five active automations, webhooks.
   * [Image BG Blurer](https://imagebgblurer.com/) -- Generate a blurred background frame for an image, using that image source as the background blur, for Notion, Trello, Jira, and more tools
   * [Webstudio](https://webstudio.is/) -- Open-source alternative to Webflow. The free plan offers unlimited websites on their domain. Five websites with custom domains. Ten thousand page views/month. 2 GB asset storage.
   * [Nappy](https://nappy.co/) -- Beautiful photos of Black and Brown people, for free. For commercial and personal use.
+  * [Syncfusion UI Component](https://www.syncfusion.com/) - A Comprehensive Suite of UI Controls and Design Components for Modern Web, Mobile, and Desktop Applications.
```

Comment:
please state what's free and be sure to have read the top of the readme for guidance

Reply:
As you mentioned I have included the details of Free community license.